docs/source/api_reference.rst

```diff
@@ -19,8 +19,10 @@ WoodworkTableAccessor
     WoodworkTableAccessor.init
     WoodworkTableAccessor.loc
     WoodworkTableAccessor.logical_types
+    WoodworkTableAccessor.metadata
```

Comment:
I believe we also need entries in the table and column schemas for the properties that were added.

Reply:
Done!